packages/fx-core/src/common/template-utils/templatesActions.ts

```diff
@@ -215,6 +216,24 @@ export async function scaffoldFromTemplates(
   context: ScaffoldContext,
   actions: ScaffoldAction[] = defaultActionSeq
 ): Promise<void> {
+  if (!context.group) {
+    throw new Error(missKeyErrorInfo("group"));
```

Comment:
This error could not be handled by context.onActionError, which may break the design of template utils.

Reply:
`This error could not be handled by context.onActionError`  - An assumption of this statement is: this error should be handled by `context.onActionError`, but it may not be true because `context.onActionError` is to handle errors throwed by Actions and here the errors are out of the Actions execution loop.

One thing I'm not so sure is whether we need to precheck these three fileds here, maybe undefined is acceptable in the very beginning of this `scaffoldFromTemplates` function.